doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1118,6 +1118,16 @@ KRKNWK-11666: CLI command ``hap services`` returns incorrect results
 KRKNWK-11365: HAP tool's ``provision`` command freezes
   This issue happens on macOS when an EUI argument is not passed in attempt to read EUI from a connected board.
 
+Google Fast Pair Service
+========================
+
+The issues in this section are related to the :ref:`bt_fast_pair_readme`.
```

Comment:
This issue is relevant for whole Fast Pair service. However, we apply workaround only for sample.

Reply:
Since the issue is relevant for whole Fast Pair service, then maybe we can move it there and inform in workaround that user needs to disable given Kconfig in the app configuration?

Reply:
I wonder if the section shouldn't be placed under Bluetooth LE (maybe we can expand the section with issues related to Bluetooth host)?

Reply:
btw. Since the issue is not yet in any NCS release then maybe it can be fixed in time (so that known issue will not be needed)? Alternatively we may add a build warning instead for the configuration that may trigger it (e.g. in Fast Pair source files).

Reply:
Since the issue is tagged with a future release, I agree and also checked with Greg, we could leave this for now and add this known issue (under integration section) towards the end of the release, if the issue is not fixed by then. 

For workaround, we have the release note entry to track. Hope this works.

Reply:
Discussed it offline. We will add known issue (if still needed) during NCS 2.5 release process. We won't add build warning.